src/components/AvatarWithImagePicker.tsx

```diff
@@ -287,11 +287,12 @@ function AvatarWithImagePicker({
 
     return (
         <View style={StyleSheet.flatten([styles.alignItemsCenter, style])}>
-            <View style={[styles.pRelative, avatarStyle]}>
+            <View style={styles.w100}>
```

Comment:
Have you QA'd this change on both the Initial Settings Page and the Workspace Profile Page?